tools/cvm-image-rewriter/README.md

```diff
@@ -46,28 +46,28 @@ There are following customization plugins in Pre-Stage providing customization t
 | 03-netplan | Customize the netplan.yaml | N |
 | 04-user-authkey | Add auth key for user login instead of password | N |
 | 05-readonly-data | Fix some file permission to ready-only | N |
-| 07-install-mvp-guest | Install MVP TDX guest kernel | Y |
-| 08-device-permission | Fix the permission for device node | Y |
-| 09-ccnp-uds-directory-permission | Fix the permission for CCNP UDS directory | Y |
+| 06-install-tdx-guest-kernel | Install MVP TDX guest kernel | Y |
+| 07-device-permission | Fix the permission for device node | Y |
+| 08-ccnp-uds-directory-permission | Fix the permission for CCNP UDS directory | Y |
 | 60-initrd-update | Update the initrd image | N |
 | 98-ima-enable-simple | Enable IMA (Integrity Measurement Architecture) feature | N |
 
 ### 2.2 Design a new plugin
 
-A plugin is put into the directory of [`pre-stage`](/tools/cvm-image-rewriter/pre-stage/),
+A plugin is put into the directory of [`plugins`](/tools/cvm-image-rewriter/plugins/),
 with the number as directory name's prefix. So the execution of plugin will be
-dispatched according to number sequence for example `99-test` is the final one.
+dispatched according to number sequence for example `99-byebye` is the final one.
 
 A plugin includes several customization approaches:
 
 1. File override: all files under `<plugin directory>/files` will be copied the
 corresponding directory in target guest image.
-2. Pre-stage execution on the host: the `<plugin directory>/host_run.sh` will be
+2. Pre-stage execution on the host: the `<plugin directory>/pre-stage/host_run.sh` will be
 executed before cloud-init stage
 3. cloud-init customization: please put the config yaml into `<plugin directory>/cloud-init/cloud-config`,
 and put the scripts to `<plugin directory>/cloud-init/x-shellscript`
 
-Please refer [the sample plugin](/tools/cvm-image-rewriter/pre-stage/99-test/).
+Please refer [the sample plugin](/tools/cvm-image-rewriter/plugins/97-test/).
 
 ## 3. How to Run the tool
```

Comment:
I would change sentence 2.

"This tool can run on bare metal or within a virtual machine using nesting as detailed in section 3.4"

Reply:
Is it worth mentioning this runs on debian 10?  That seems like an OS no one uses very much.  I could be wrong.